packages/core-mobile/app/screens/defi/DeFiProtocolDetails.tsx

```diff
@@ -102,7 +103,7 @@ export const DeFiProtocolDetails = (): JSX.Element => {
   }
 
   return (
-    <View style={styles.container}>
+    <SafeLowerAreaView sx={styles.container}>
```

Comment:
is the WalletScreenStack too complex to apply SafeLowerAreaView? I see that you are doing it manually for some of its screens

Reply:
It's because the `DrawerView` is in it and needs to bleed out as well.

Wrapping screen stack was a way to make this change not to big at the moment, but eventually it would be great if these safe area views moved down the hierarchy to the view that needs it.